espnet/bin/asr_recog.py

```diff
@@ -257,6 +257,13 @@ def get_parser():
         default=0.999,
         help="Threshold probability for CTC output",
     )
+    # quantize model related
+    parser.add_argument(
+        "--quantize-model",
+        type=bool,
```

Comment:
I wonder if this could be string instead of bool to specify not only qint8 but also qint16, qint32, etc. How about this?
```python
# asr_recog.py
parser.add_argument(
    "--quantize-dtype", type=str, default=None, help="dtype for torch.quantization.quantize_dynamic")
# recog.py and asr.py
if args.quantize_dtype:
    logging.info("Use quantized model for decoding")
    dtype = getattr(torch, args.quantize_dtype)
    model = torch.quantization.quantize_dynamic(model, {torch.nn.Linear}, dtype=dtype)
```

Reply:
thanks